napari/layers/_base_layer/model.py

```diff
@@ -50,9 +48,7 @@ class Layer(VisualWrapper, ABC):
 
     def __init__(self, central_node, name=None):
         super().__init__(central_node)
-        self._selected = False
-        self._qt_properties = None
-        self._qt_controls = None
+        self._selected = True
```

Comment:
what prompted this change?

Reply:
it used to be set in the view, but should have been set here instead